Sources/Dispatcher.swift

```diff
@@ -498,3 +498,481 @@ public extension CatchMixin where T == Void {
         return recover(on: dispatcher, policy: policy, body)
     }
 }
+
+//////////////////////////////////////////////////////////// Cancellation
+
```

Comment:
All of the 'Cancellation' code in Dispatcher can be removed if we decide we don't want/need strict compatibility with the non-cancellable classes (these are all wrappers for the Dispatcher change).

Reply:
Lmk if I should delete this code.  Not sure what compatibility is desired here.